cfgov/unprocessed/css/molecules/full-width-text.less

```diff
@@ -6,10 +6,15 @@
       markup: |
         <div class="o-full-width-text-group">
             <div class="m-full-width-text">
-              <ul>
-                <li>Gray Well</li>
-                <li>Well's content</li>
-              </ul>
```

Comment:
Sorry, I didn't mean just the list, I meant all of the markup. Before we were modifying the lists, but now that we're not doing anything except changing the font-family with a modifier, it seems odd to include any details other than those for the modifier.